backstory/src/test/java/com/google/sps/servlets/PromptManagerTest.java

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 Google LLC
```

Comment:
NIT: Copyright 2020 Google LLC (I always forget to change this too).

Reply:
Done